### DISCUSSION TITLE
Tango board

### DISCUSSION THREAD

**Discussion by kzali** [multigcs/LinuxCNC-RIO#22] [Q&A] [answered]
Hi Oliver
I had 2 PCB's populated with components from JLCPCB.  They have populated a resistor or a diode across pins 5 and 7 of PMOD1.  Cant find it in the schematics.  I can remove it,  I don't understand what PMOD1 is for.   I presume its for expansion 
image: https://drive.google.com/file/d/1XzmRCHtgt0va_CWWgjXDTOYHbClTo75q/view?usp=drive_link

Can you advise 
Thanks

**Comment by multigcs** [comment]
Can not see your image, access deny :(

PMOD is a standard interface connector for FPGA development: https://digilent.com/reference/_media/reference/pmod/pmod-interface-specification-1_2_0.pdf

its is free usable on the tangoboard (3.3V only !!!).

What did you pay for the boards?

**Comment by TurBoss** [comment]
morning, I can see the image duno why

here its his upload

![rio-funny component](https://github.com/multigcs/LinuxCNC-RIO/assets/184088/c5af3f3a-b70f-4b2d-b7e5-f065eeca3b97)

**Reply by kzali** [reply]
Sorry hadn't set the permissions.  Git was not allowing me to add the image (don't know why!)  
I paid AU$16 each with all active and passive components.  Got the pcb connector pairs for 2 boards at AU$9 from Ali and had the headers already.  I would say $30 max per board.   Still have a lot of learning.  Still have to figure out the wiring but love your project.  I think its a 3nf capacitor (not sure though)
Thanks

**Comment by multigcs** [comment]
hmm, ok, no please remove.

I am not sure if my circuit diagram is perfect and all values are set with the correct values.

i use 3,3kOhm for all resistors

**Reply by kzali** [reply]
I checked your pcb design files.  This was not there and on my boards are 3.3kOhm.  So will remove that capacitor.  
Thanks

**Comment by multigcs** [comment]
btw, the price is very good !

**Reply by kzali** [reply]
JLCPCB is very good and economical.  Thanks for your commendable work

**Comment by ghost** [comment]
Looks like a pick and place eff up. lol

**Reply by kzali** [reply]
Ha ha yes Rob.  One of those boards is yours mate, since you are my only help here in Sydney (Just to play)

**Comment by kzali** [comment]
Hello Oliver

For the Tangoboard in the install.md : https://github.com/multigcs/LinuxCNC-RIO/blob/main/configs/Tangoboard/INSTALL.md

There is no config-udp.  when I look at config-mycnc.json it seems to be setup for the ethernet boards, whereas install md says to use config-udp for ethernet boards 

"or if you want to use the optional Ethernet-Board:
python3 buildtool.py configs/Tangoboard/config-udp.json"

Don't really know what I am doing wrong!  I want to use an Eth01 board.  Can you advise?

**Reply by bioname** [reply]
You need to update configuration to your own IPaddress for FPGA, and update IP address of your Linux ETH interface, both must be in the same network. For example I use 10.10.10.10 for FPGA and 10.10.10.1 for Linux side. No any default routes required.
Don't forget to rebuild everything( rio.c with Hal compile also)

**Comment by multigcs** [comment, marked as answer]
Hi kzali,
sorry, my fault, the install.md is more or less a copy of the one from the TangNano9K folder.
the default config.json in the tangoboard folder have allready UDP configured.

```
    "transport": "UDP",
    "ip": "192.168.10.14",
```

.... thank you bioname :)

**Reply by kzali** [reply]
Thank you all.  When I looked at config.json, I thought it was right.  Just wanted to make sure
cheers
